virttest/utils_test/qemu/__init__.py

```diff
@@ -488,7 +488,7 @@ def get_guest_total_mem(cls, vm):
         :return: physical memory report by guest OS in MB
         """
         if vm.params.get("os_type") == "windows":
-            cmd = "wmic ComputerSystem get TotalPhysicalMemory"
+            cmd = 'powershell -command "(Get-CimInstance -ClassName Win32_ComputerSystem).TotalPhysicalMemory"'
```

Comment:
@leidwang also hit a timeout with the following command, either executing it on the test loop or individually:
`ERROR: Timeout expired while waiting for shell command to complete: 'powershell -command "(Get-CimInstance -ClassName Win32_ComputerSystem).TotalPhysicalMemory"'    (output: '')`